src/main/java/edu/harvard/iq/dataverse/license/LicenseListingPage.java

```diff
@@ -50,6 +50,7 @@
     private List<LicenseDto> licenses = new ArrayList<>();
 
     private LicenseDto freshLicense;
+    private LicenseDto selectedLicense;
```

Comment:
I would rename this to `licenseForPreview`. `selected` suggest that you pick something from list or or some radio button